app/javascript/app/pages/climate-policies/milestones/milestones-styles.scss

```diff
@@ -0,0 +1,88 @@
+@import '~styles/ui-extends.scss';
+@import '~styles/layout.scss';
+
+$iconSize: 60px;
+$borderSize: 5px;
+$correction: 1px;
+
+.page {
+  margin-bottom: 50px;
+}
+
+.titleContainer {
+  display: flex;
+  justify-content: center;
+  background-color: $link-water;
+  border-bottom: 1px solid rgba($jakarta, 0.3);
+  color: $downriver;
+}
+
+.title {
+  padding: 30px;
+  text-transform: uppercase;
+  font-size: $font-size-large;
+  font-weight: bold;
+}
+
+.cell {
+  padding: 20px;
+  color: $downriver;
+  line-height: 1.5;
+}
+
+.statusColumn {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  font-weight: bold;
+  border-right: $borderSize solid $elephant-blue;
```

Comment:
On the designs the border has a slightly darker shade of blue, instead of `#113750` (`$elephant-blue`) it should be `#0a1941` (`$theme-color`)

![image](https://user-images.githubusercontent.com/6906348/50223903-bc460e00-039c-11e9-900e-0df65c70ae66.png)